lldb/source/Commands/CommandObjectBreakpoint.cpp

```diff
@@ -769,20 +769,26 @@ class CommandObjectBreakpointSet : public CommandObjectParsed {
 private:
   bool GetDefaultFile(Target &target, FileSpec &file,
                       CommandReturnObject &result) {
-    uint32_t default_line;
     // First use the Source Manager's default file. Then use the current stack
     // frame's file.
-    if (!target.GetSourceManager().GetDefaultFileAndLine(file, default_line)) {
+    auto file_and_line = target.GetSourceManager().GetDefaultFileAndLine();
```

Comment:
Suggestion: It's not obvious this is an optional type. Maybe name it `maybe_file_and_line`? 😄